### ISSUE TITLE
Performing edits while pattern editor not focused crashes

### ISSUE THREAD

**Issue by SuperDisk** [SuperDisk/UGE#157]
crash on transposing a semitone using the right click menu when the pattern editor doesn't have keyboard focus... "The error is: EInvalidCast with message Invalid type cast"

From madbrain

**Comment by SuperDisk** [comment]
I'm not sure how to actually reproduce this issue, but it might be that on Windows you can get the context menu to appear without having the control focused somehow. Added checks to prevent this in [97fea04](https://github.com/SuperDisk/hUGETracker/commit/97fea04e5ee186a9b5e6220df786a3dfec8ed435)
